zio-cli/shared/src/main/scala/zio/cli/Options.scala

```diff
@@ -24,6 +24,16 @@ import zio.cli.HelpDoc.Span._
 
 import scala.collection.immutable.Nil
 
+final case class ValidationError(validationErrorType: ValidationErrorType, error: HelpDoc) {
```

Comment:
Could we move these two types to `validation.scala`?

Reply:
I've moved the Validation classes into a separate file now.